### PR TITLE
Benchmarks: Build Pipeline - Support rocm cmake build

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Support rocm cmake build. 

**Major Revision**
- Add  some envs in rocm_common.cmake to support rocm cmake build.

### CODE REVIEW

File: superbench/benchmarks/micro_benchmarks/rocm_common.cmake [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+else()
+set(HIP_PATH $ENV{HIP_PATH})
+endif()
+
+if(EXISTS ${HIP_PATH})
+    # Search for hip in common locations
+    list(APPEND CMAKE_PREFIX_PATH ${HIP_PATH} ${ROCM_PATH})
+    set(CMAKE_MODULE_PATH "${HIP_PATH}/cmake" ${CMAKE_MODULE_PATH})
+    set(CMAKE_MODULE_PATH "${HIP_PATH}/lib/cmake/hip" ${CMAKE_MODULE_PATH})
+endif()
```
Comment (cp5555) [inline superbench/benchmarks/micro_benchmarks/rocm_common.cmake:23]: Add new line.

File: superbench/benchmarks/micro_benchmarks/rocm_common.cmake [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Set ROCM_PATH
+if(NOT DEFINED ENV{ROCM_PATH})
+set(ROCM_PATH /opt/rocm)
```
Comment (cp5555) [inline superbench/benchmarks/micro_benchmarks/rocm_common.cmake:6]: Do we need to add space for this?

File: superbench/benchmarks/micro_benchmarks/rocm_common.cmake (hunk 1 above)
Comment (yukirora) [reply superbench/benchmarks/micro_benchmarks/rocm_common.cmake]: ok, changed

File: superbench/benchmarks/micro_benchmarks/rocm_common.cmake (hunk 2 above)
Comment (yukirora) [reply superbench/benchmarks/micro_benchmarks/rocm_common.cmake]: ok, changed

**Review by cp5555 — APPROVED** [review]

**Review by abuccts — APPROVED** [review]

**Pull request merged**